c/example-4-piggybank/README.md

```diff
@@ -0,0 +1,49 @@
+# Piggybank
+
+This contract has two methods. The first method (`initialize_target`)
+specifies the target amount of money that should be collected by
+the contract.
+
+The second method (`transfer`) transfers out almost all the money from 
+the account (reduced by fixed `MESSAGE_COST` amount, a fixed constant in
+the contract C file), but this will happen only if the contract accumulated 
+at least target amount of money.
+
+## Methods
+
+### Method `initialize_target`
+#### Input values
+##### Argument `target`
+* ABI type: `uint64`
+* C type: `unisnged`
+* Description: specifies target amount of money.
+
+#### Output values
+None
+
+#### Notes
+This method initializes the contract with the target amount of money.
```

Comment:
This method initializes the contract with the target amount of money.
The method may be called again with a greater target amount to
increase it. However, if the new target amount is smaller than the
previous one, the method will fail.